spec.html

```diff
@@ -5490,7 +5483,7 @@ <h1>IsRegExp ( _argument_ )</h1>
 
     <emu-clause id="sec-isstringprefix" aoid="IsStringPrefix">
       <h1>IsStringPrefix ( _p_, _q_ )</h1>
-      <p>The abstract operation IsStringPrefix determines if String _p_ is a prefix of String _q_.</p>
+      <p>The abstract operation IsStringPrefix takes arguments _p_ (a String) and _q_ (a String). It determines if _p_ is a prefix of _q_ It performs the following steps:</p>
```

Comment:
We should look into giving aliases like these more meaningful names. This is not a generic enough operation to warrant completely meaningless names. @tc39/ecma262-editors What do you think?

Reply:
I think that we should apply that sensibility to new abstract ops, but I don't think the churn is often likely to be worth it.